### PR TITLE
 License.txt: Include full official LGPL 2.1 .txt file

### PR DESCRIPTION
**Pull request by paramat**
Copy-paste and only remove form-feed characters.
////////////////////////////

### CODE REVIEW

**Comment by rubenwardy** [comment]
:+1:

**Comment by sofar** [comment]
I wouldn't manually reformat this. Please keep it exactly the same as how it comes from fsf.org

https://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt

**Comment by paramat** [comment]
Ok i will add the missing stuff at the end, but we can't have tabs where spaces should be or form feed characters.

**Comment by sofar** [comment]
The file is `.txt`, not `.md`. The tabs are all fine. We should probably just include the lgpl as COPYING instead, and not include it as part of LICENSE.txt. That way it can be verbatim and unmodified as well.

**Comment by paramat** [comment]
Yeah copying the txt document is fine, it uses spaces for indentation, i have just removed the form feed characters.

**Comment by sofar** [comment]
yup, that looks fine.

File: LICENSE.txt [hunk 1]
```diff
@@ -472,5 +472,48 @@ FAILURE OF THE LIBRARY TO OPERATE WITH ANY OTHER SOFTWARE), EVEN IF
 SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH
 DAMAGES.
 
-   END OF TERMS AND CONDITIONS
-
+                     END OF TERMS AND CONDITIONS
+
+           How to Apply These Terms to Your New Libraries
```
Comment (sfan5) [inline LICENSE.txt]: why include this section?

Comment (Calinou) [inline LICENSE.txt]: [The FSF recommends including it, in fact, you may not be even allowed to remove it at all.](https://www.gnu.org/licenses/gpl-faq.en.html#GPLOmitPreamble)

**Comment by paramat** [comment]
b6889134d34c7aa933bd1e4a808605ae2228bc97

**Pull request closed without merging**